seasons.py

```diff
@@ -1,21 +1,17 @@
-def season (month):  # remove whitespace `def season(month):`
+def season(month):  # remove whitespace `def season(month):`
```

Comment:
remove my comments please
```suggestion
def season(month):
```

Reply:
Please add type hints:
```suggestion
def season(month: int) -> str:
```